### ISSUE TITLE
Enabling firewalld daemon, stops Haproxy logging

### ISSUE THREAD

**Issue by ben-hart-jamf** [firewalld/firewalld#883]
**What happened**: Installed firewalld via apt, installation started the service which then halted haproxy log file input

**What you expected to happen**: I expected the firewalld daemon to not be started automatically, but either way, also expected no logging/syslog impacts.

**How to reproduce it (as minimally and precisely as possible)**: Installed haproxy/focal-updates 2.0.13, verify normal global log config is:
```
global
        log /dev/log    local0
#        log /dev/log    local1 notice
	log 127.0.0.1	local1
        chroot /var/lib/haproxy
```
Start the service, tail /var/log/haproxy.log for activity. Install Firewalld via APT, once complete restart haproxy then tail log file.
**Anything else we need to know?**:

**Environment**: Ubuntu 20.04.03 LTS, vSphere 7
- Firewalld Version (if Fedora based `dnf info firewalld` or commit hash if developing from git `git log -n1 --format=format:"%H"`): 0.8.2-1 all
- Firewalld Backend (`cat /etc/firewalld/firewalld.conf | grep FirewallBackend`): iptables
- OS (e.g: `cat /etc/os-release`): 
```
- NAME="Ubuntu"
VERSION="20.04.3 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.3 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```
- Others:
I have tried changing the Haproxy default syslog facilities from 0 ,1 to 4,5 on the off chance FirewallD is using the same. I noticed /var/log/firewalld.log empty, and no custom syslog in /etc/rsyslog.d

**Comment by ben-hart-jamf** [comment]
So, I think I've discovered what happened. So in the haproxy.cfg  snippet above, the line for 127.0.0.1 was added for a reason. I swear. Right now, I can't find the page nor the reason why. I mention that because if I comment out that line, bounce all related daemons Haproxy start logging correctly (as per /etc/rsyslog.d/49-haproxy.conf).
On this system, Rsyslog is not setup as a syslog server or recipient so port UDP/514 isn't open, and there were no log entries from FirewallD about this.. at least in the Kern entries about connections being rejected/allowed/etc. 
But, I also don't believe I'm seeing all the logs for FirewallD.
